src/libs/Localize/index.ts

```diff
@@ -46,7 +46,13 @@ function init() {
     }, {});
 }
 
-type PhraseParameters<T> = T extends (...args: infer A) => string ? A : never[];
+type PluralFormPhrase = Record<string, string>;
+type PhraseParameters<T> = T extends (args: number | Record<string, string | number | undefined>) => string
```

Comment:
Actually, wouldn't the old version of this type work?
```suggestion
type PhraseParameters<T> = T extends (...args: infer A) => string ? A : never[];
```